google/cloud/storage/internal/retry_client_test.cc

```diff
@@ -17,6 +17,11 @@
 #include "google/cloud/storage/testing/mock_client.h"
 #include "google/cloud/testing_util/chrono_literals.h"
 #include "google/cloud/testing_util/status_matchers.h"
+#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
```

Comment:
nit: we should only need the `#ifdef` around includes of `<opentelemetry/...>`. Our headers should have this conditional logic built in.

Reply:
Fixed.